default.nix

```diff
@@ -137,6 +136,23 @@ let
   deb-packages = moveDerivations "deb-packages" (map buildDeb packageDescs);
   rpm-packages = moveDerivations "rpm-packages" (map buildRpm packageDescs);
 
+  releaseFile = writeTextFile {
+    name = "release-notes.md";
+    text = ''
+      Automatic release on ${builtins.substring 0 8 timestamp}
+
+      This release contains assets based on [${babylonnet.rev} revision](https://gitlab.com/tezos/tezos/tree/${babylonnet.rev}) of babylonnet branch and
+      [${mainnet.rev} revision](https://gitlab.com/tezos/tezos/tree/${mainnet.rev}) of mainnet branch from [tezos repository](https://gitlab.com/tezos/tezos/).
+
+      For more information about release assets see [README section](https://github.com/serokell/tezos-packaging#obtain-binaries-or-packages-from-github-release).
```

Comment:
I think for actual releases (not auto pre-releases) the link should be https://github.com/serokell/tezos-packaging/blob/202001221235/README.md#obtain-binaries-or-packages-from-github-release. So I propose to change it to https://github.com/serokell/tezos-packaging/blob/autorelease/README.md#obtain-binaries-or-packages-from-github-release here and add a comment saying that one should change `autorelease` to a new tag when one makes a release.